.github/workflows/build.yml

```diff
@@ -78,12 +78,10 @@ jobs:
         - if: steps.changes.outputs.ruby == 'true'
           name: run zeitwerk:check
           run: bin/rails zeitwerk:check
-        - continue-on-error: true
-          if: steps.changes.outputs.ruby == 'true'
+        - if: steps.changes.outputs.ruby == 'true'
```

Comment:
The specs should all pass now on Rails 6.1 so I'm removing the exception that let future steps run even on failure.